### DISCUSSION TITLE
Webcam putting radius needs more resolution

### DISCUSSION THREAD

**Discussion by wittsjo** [springbok/MLM2PRO-GSPro-Connector#153] [General]
I can't get good enough accuracy on the putting speed (radius), if i set it at 3 it's to slow, 4 is too fast etc. Is there a command to set it to a value with a decimal or so? I recommend at least adding half steps on that scale.

Thanks for a great software.

**Comment by wonder99** [comment]
This one is out of our control--it's controlled by the underlying webcam putting by alleexx.
https://github.com/alleexx/cam-putting-py/blob/main/ball_tracking_setup.py

![image](https://github.com/user-attachments/assets/d004540f-f924-4e5e-98a1-0eb58803081c)
